exportReport/FileChooser/ReportFileView.java

```diff
@@ -0,0 +1,102 @@
+package org.zaproxy.zap.extension.exportReport.FileChooser;
+
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ * 
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License. 
+ * You may obtain a copy of the License at 
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0 
+ *   
+ * Unless required by applicable law or agreed to in writing, software 
+ * distributed under the License is distributed on an "AS IS" BASIS, 
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+ * See the License for the specific language governing permissions and 
+ * limitations under the License. 
+ * This file is based on the Paros code file ReportLastScan.java
+ */
+
+import java.io.File;
+import javax.swing.Icon;
+import javax.swing.filechooser.*;
```

Comment:
Same as earlier, and likely applicable throughout. It would be great if wildcard imports could be replaced by something more specific.